lib/rucio/core/did_meta_plugins/__init__.py

```diff
@@ -16,157 +16,194 @@
 # Authors:
 # - Aristeidis Fkiaras <aristeidis.fkiaras@cern.ch>, 2020
 # - Benedikt Ziemons <benedikt.ziemons@cern.ch>, 2020-2021
+# - David Población Criado <david.poblacion.criado@cern.ch>, 2021
+# - Rob Barnsley <robbarnsley@users.noreply.github.com>, 2021-2022
```

Comment:
A lot of the changes here are just house keeping - adding comments and amending docstrings etc.